QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java

```diff
@@ -367,23 +367,19 @@ private void updateButtonState(SendButtonState buttonState) {
             float barRotationTarget2 = mButtonState == SendButtonState.ATTACH ? 90 : 135;
             float barTranslationTarget2 = mButtonState == SendButtonState.SEND ? translation : 0;
 
-            if (barRotation1 != barRotationTarget1) {
                 ObjectAnimator.ofFloat(mButtonBar1, "rotation", barRotation1, barRotationTarget1)
                         .setDuration(ANIMATION_DURATION)
                         .start();
                 ObjectAnimator.ofFloat(mButtonBar2, "rotation", barRotation2, barRotationTarget2)
                         .setDuration(ANIMATION_DURATION)
                         .start();
```

Comment:
Now that the code isn't nested, they're wrongly indented

Can you remove the extra indentation?